OrbitCore/LinuxEventTracer.cpp

```diff
@@ -10,6 +10,7 @@
 
 #include <vector>
 
+#include "BpfTraceVisitor.h"
```

Comment:
Should we have a TODO (or an issue for tracking) to remove bpftrace stuff?

Reply:
I'm removing the references to BpfTrace in the next PR, for now I'll add a TODO.